relay-server/src/actors/project_cache.rs

```diff
@@ -86,16 +86,28 @@ impl ProjectCache {
     fn get_or_create_project(&mut self, project_key: ProjectKey) -> &mut Project {
         metric!(histogram(RelayHistograms::ProjectStateCacheSize) = self.projects.len() as u64);
 
-        let config = self.config.clone();
+        let config = &self.config;
```

Comment:
I assume this change is for some performance improvement, on potentially not cloning the config in the section below?

Reply:
Yep, potentially prevents an unnecessary clone.